Source/WebCore/html/canvas/WebGLBuffer.cpp

```diff
@@ -45,7 +45,6 @@ WebGLBuffer::WebGLBuffer(WebGLRenderingContextBase& ctx)
     : WebGLSharedObject(ctx)
 {
     setObject(ctx.graphicsContextGL()->createBuffer());
-    clearCachedMaxIndices();
```

Comment:
Should this `clearCachedMaxIndices` call be left under `#if !USE(ANGLE)`?

Reply:
to avoid ifdeffing, the structure being initialised by clearCachedMaxIndices is initialised as {}, zero-initialised.